presidio-image-redactor/presidio_image_redactor/bbox.py

```diff
@@ -0,0 +1,130 @@
+from typing import List, Tuple
+
+
+class BboxProcessor:
+    """Common module for general bounding box operators."""
+
+    @staticmethod
+    def get_bboxes_from_ocr_results(ocr_results: dict) -> List[dict]:
+        """Get bounding boxes on padded image for all detected words from ocr_results.
+
+        :param ocr_results: Raw results from OCR.
+        :return: Bounding box information per word.
+        """
+        bboxes = []
+        item_count = 0
+        for i in range(len(ocr_results["text"])):
+            detected_text = ocr_results["text"][i]
+            if detected_text:
+                bbox = {
+                    "left": ocr_results["left"][i],
+                    "top": ocr_results["top"][i],
+                    "width": ocr_results["width"][i],
+                    "height": ocr_results["height"][i],
+                    "conf": float(ocr_results["conf"][i]),
+                    "label": detected_text,
+                }
+                bboxes.append(bbox)
+                item_count += 1
+
+        return bboxes
+
+    @staticmethod
+    def get_bboxes_from_analyzer_results(analyzer_results: list) -> List[dict]:
+        """Organize bounding box info from analyzer results.
+
+        :param analyzer_results: Results from using ImageAnalyzerEngine.
+
+        :return: Bounding box info organized.
+        """
+        bboxes = []
+        for i in range(len(analyzer_results)):
+            result = analyzer_results[i].to_dict()
+
+            bbox_item = {
+                "entity_type": result["entity_type"],
+                "score": result["score"],
+                "left": result["left"],
+                "top": result["top"],
+                "width": result["width"],
+                "height": result["height"],
+            }
+            bboxes.append(bbox_item)
+
+        return bboxes
+
+    @classmethod
+    def format_bboxes_for_dicom(
```

Comment:
Is this only for dicom? if we call it `remove_padding_from_results` or something similar, then we could use it also for non-dicom cases, right?